### PR TITLE
Register API to. my domain (Zoom). DISCORD & GITHUB verification system.

### PR DESCRIPTION
**Pull request by TheOldZoom**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [ ] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/api.zoom.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    "description": "API for my images & more aka a CDN",
+    "repo": "https://github.com/theoldzoom/api",
+    "owner": {
+        "username": "theoldzoom",
+        "email": "theoldzoom@gmail.com",
+        "twitter": "theoldzoom",
+        "discord": "theoldzoom"
+    },
+    "record": {
+        "CNAME": "theoldzoom.github.io/api/"
```
Comment (21Z) [inline domains/api.zoom.json:11]: `CNAME` can't contain `/`

**Review by 21Z — CHANGES_REQUESTED** [review]

**Review by triach-rold — APPROVED** [review]

File: domains/_discord.zoom.is-a.dev.json
Comment (CuteDog5695) [inline domains/_discord.zoom.is-a.dev.json:1]: remove .is-a.dev from the json file please

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
should be fine now

**Comment by TheOldZoom** [comment]
Thanks you @CuteDog5695

**Pull request merged**